src/aura/CRLP_RollupsContainer/CRLP_RollupsContainerHelper.js

```diff
@@ -63,13 +96,45 @@
 
     /* @description: toggles a modal popup and backdrop
     */
-    toggleFilterRuleModal: function(cmp){
+    toggleFilterRuleModal: function(cmp) {
         var backdrop = cmp.find('backdrop');
         $A.util.toggleClass(backdrop, 'slds-backdrop_open');
         var modal = cmp.find('modaldialog');
         $A.util.toggleClass(modal, 'slds-fade-in-open');
     },
 
+    /**
+    * @description: queries the rollups to update filter group name references
+    */
+    requeryRollups: function(cmp) {
+        var action = cmp.get("c.getRollupDefinitions");
+
+        //requery rollup records
+        action.setCallback(this, function(response) {
+            console.log('requeried rollup definitions');
+            var state = response.getState();
+            if (state === "SUCCESS") {
+                var rollupList = JSON.parse(JSON.stringify(response.getReturnValue()));
+
+                cmp.set("v.rollupList", rollupList);
+                cmp.set("v.cachedRollupList", rollupList);
+            }
+            else if (state === "ERROR") {
+                var errors = response.getError();
+                if (errors) {
+                    if (errors[0] && errors[0].message) {
+                        console.log("Error message: " +
```

Comment:
As we're getting closer to a production release, does it make sense to start adding toasts for unexpected errors in apex callbacks so the user has some understanding of why the page may not be working as expected?

Reply:
Yes, I agree with that. I added one where you had asked for it, but I think adding them in for these is a good idea (and will help us/support with debugging too).